_dev/apps/ui/src/store/modules/campaigns/actions.ts

```diff
@@ -325,6 +325,7 @@ export default {
     query.append('limit', state.campaigns.request.numberOfCampaignsPerPage);
     query.append('startDate', state.reporting.request.dateRange.startDate);
     query.append('endDate', state.reporting.request.dateRange.endDate);
+    query.append('currency_code', rootState.googleAds.accountChosen.currencyCode);
```

Comment:
@tberger-prestashop This variable is based from the "recommanded-budget" call, but does not match the case of other variabes. Let me know if I need to change it.